linkis-spring-cloud-services/linkis-service-gateway/linkis-gateway-core/src/main/scala/org/apache/linkis/gateway/config/GatewayConfiguration.scala

```diff
@@ -63,4 +63,7 @@ object GatewayConfiguration {
 
   val DSS_QUERY_WORKSPACE_SERVICE_NAME = CommonVars("wds.dss.query.workspace.service", "dss-framework-project-server")
   val USER_WORKSPACE_REFLESH_TIME  = CommonVars("wds.linkis.user.workspace.reflesh.time", 10)
+
+  // Use regex to match against URLs, if matched, let them pass anyway(even if not currently logged in)
+  val GATEWAY_NO_AUTH_URL_REGEX = CommonVars("wds.linkis.gateway.no.auth.url.regex", ".*visualis.*share.*")
```

Comment:
Hello, is it better to consider configuring multiple regulars?

Reply:
Not sure if I understand your meaning. Because REGEX already supported the OR relation(So you could match  against multiple URLs with just one pattern), which eliminated the need for "configuring multiple regulars"

Could you provide some examples?

Reply:
> Not sure if I understand your meaning. Because REGEX already supported the OR relation(So you could match against multiple URLs with just one pattern), which eliminated the need for "configuring multiple regulars"
> 
> Could you provide some examples?

Yes, that's what it means, you can provide an example in the comments

Reply:
With "()" and "|" you can express the "OR" relation in REGEX, for instance:

(.*visualis.*|.*dss.*) will MATCH URLs containing visualis OR  dss, the following URLS with be matched:
1. http://linkis.org/jd/visualis/vis
2. http://linkis.org/jd/dss/vis

Other URLs will NOT be matched.